### PR TITLE
Adding wrapper for native geocoding

### PR DESCRIPTION
**Pull request by RobinCsl**
Please, @lukewalczak and @ferrannp, give me your feedback on the code as Kotlin and Objective C are not my primary languages... 😄

### CODE REVIEW

**Comment by RobinCsl** [comment]
I added WIP as it needs to be reviewed by both @lukewalczak and @ferrannp before this can be merged.

File: ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#if __has_include("RCTBridgeModule.h")
+#import "RCTBridgeModule.h"
+#else
+#import "React/RCTBridgeModule.h"
+#endif
```
Comment (lukewalczak) [inline ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.h]: You don't need if-else. `#import <React/RCTBridgeModule.h>` is enough.

File: ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.m [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_get_main_queue();
+}
+
+RCT_EXPORT_MODULE()
+
+#pragma mark -
+#pragma mark format RCT callback response
```
Comment (lukewalczak) [inline ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.m]: You don't need pragma mark - it's needed (but optional) to mark section in the class. In your case you just exporting module with the method - pragma mark is redundant.

File: ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.m [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                                   @"longitude": @(mapItem.placemark.coordinate.longitude)} forKey:@"location"];
+        
+        [RCTResponse addObject:formedLocation];
+    }
+    
+    return [RCTResponse copy];
+}
+
+#pragma mark -
+#pragma mark RCT Exports
```
Comment (lukewalczak) [inline ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.m]: Same as above.

**Comment by tbergq** [comment]
Then maybe this PR should be closed @RobinCsl?

**Comment by tbergq** [comment]
Will be added as a separate repo

**Comment by RobinCsl** [comment]
https://github.com/kiwicom/react-native-reverse-geocode

**Pull request closed without merging**